hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -233,7 +235,8 @@ private void handlePartialStart(HttpPostCommand command) throws UnsupportedEncod
             if (!(groupConfig.getName().equals(groupName) && groupConfig.getPassword().equals(groupPass))) {
                 res = res.replace("${STATUS}", "forbidden");
             } else {
-                boolean success = node.getNodeExtension().triggerPartialStart();
+                final InternalHotRestartService hotRestartService = node.getNodeExtension().getInternalHotRestartService();
+                boolean success = hotRestartService != null && hotRestartService.triggerPartialStart();
```

Comment:
I guess we don't have many of these checks before so it was ok. For now, I am ok with both approaches. We can introduce a dummy hot restart service.